### PR TITLE
Have Appveyor push a x86_64-pc-windows-gnu build when a tag is pushed

### PR DESCRIPTION
**Pull request by phrohdoh**

### CODE REVIEW

**Comment by phrohdoh** [comment]
```
    Finished debug [optimized + debuginfo] target(s) in 100.57 secs
     Running `C:\projects\openaoe\target\debug\open_aoe-f0430c1aef0c5323.exe`
error: test failed
Command exited with code -1073741515
```

http://stackoverflow.com/a/22040708/3816796 claims that:
`-1073741515 == 0xC0000135 == STATUS_DLL_NOT_FOUND`

**Comment by janisozaur** [comment]
Hi, perhaps you may find this useful: https://www.appveyor.com/docs/how-to/rdp-to-build-worker/

**Comment by phrohdoh** [comment]
@janisozaur Thanks!

File: appveyor.yml [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+      CHANNEL: nightly
+      SDL_TARGET: x86_64-w64-mingw32\lib
+      SDL_COPY_PATTERN: libSDL2*
+
+install:
+  - ps: Start-FileDownload "https://static.rust-lang.org/dist/rust-nightly-${env:TARGET}.exe"
+  - rust-nightly-%TARGET%.exe /VERYSILENT /NORESTART /DIR="C:\Program Files (x86)\Rust"
+  - SET PATH=%PATH%;C:\Program Files (x86)\Rust\bin
+  - SET PATH=%PATH%;C:\msys64\mingw64\bin;C:\msys64\usr\bin
+  - SET PATH=%PATH%;SDL2-2.0.4\${env:SDL_TARGET}\bin
```
Comment (phrohdoh) [inline appveyor.yml]: This PATH cannot be relative but I do not know the absolute path of the current working directory.

**Comment by phrohdoh** [comment]
@angered-ghandi I can get this to not fail if I remove `test_script`, but I am not able to test deployment/upload because I cannot push tags.

**Pull request merged**